### DISCUSSION TITLE
Reverse proxy that only logs slow responses (newbie)

### DISCUSSION THREAD

**Discussion by mannion007** [mitmproxy/mitmproxy#6274] [Q&A]
Disclaimer: completely new to mitm proxy!

If I were to utilise mitm proxy between 2 backend services where I see occasionally extremely slow responses from service B to service A - would it be possible to configure mitm to dump diagnostic data where say, the time to last byte was > 5s? In other cases, just be a completely transparent proxy?

**Comment by ThomasBucaioni** [comment]
Hello,
You could make your own python script as an add-on, for example this one logs events: https://github.com/mitmproxy/mitmproxy/blob/main/examples/addons/log-events.py
